Cargo.toml

```diff
@@ -94,6 +94,10 @@ log = "0.4"
 console = { version = "0.14", optional = true }
 fern = { version = "0.6", optional = true }
 itertools = "0.10"
+# Used for creating global mutex-wrapped caches
+once_cell = "1.10"
+# Faster alternative to std::mutex
+parking_lot = "0.12"
```

Comment:
the parking_lot mutex is [not always faster anymore](https://github.com/rust-lang/rust/pull/95035#issuecomment-1073966631) at least on linux since the futex-based mutex landed in std. it might worth further investigation for our specific workload.

Reply:
It looks like this is coming in 1.62. Maybe worth keeping parking_lot for the time being and then reevaluating? Or maybe we just rip it out, honestly tile creation is such a small portion of the time in rav1e that I doubt it matters.

Reply:
I guess preparing a benchmark and see which between usync, parking_lot and std work best for us.